contracts/handlers/ERC1155Handler.sol

```diff
@@ -106,11 +107,13 @@ contract ERC1155Handler is IHandler, ERCHandlerHelpers, ERC1155Safe, ERC1155Hold
         @notice Sets {_resourceIDToContractAddress} with {contractAddress},
         {_tokenContractAddressToTokenProperties[tokenAddress].resourceID} with {resourceID} and
         {_tokenContractAddressToTokenProperties[tokenAddress].isWhitelisted} to true for {contractAddress} in ERCHandlerHelpers contract.
+        Reverts if {contractAddress} doesn't suppor {_INTERFACE_ID_ERC1155}.
         @param resourceID ResourceID to be used when making deposits.
         @param contractAddress Address of contract to be called when a deposit is made and a deposited is executed.
         @param args Additional data to be passed to specified handler.
      */
     function setResource(bytes32 resourceID, address contractAddress, bytes calldata args) external onlyBridge {
+        require(contractAddress.supportsInterface(_INTERFACE_ID_ERC1155), "token does not support IERC1155");
```

Comment:
```suggestion
        require(contractAddress.supportsInterface(type(IERC1155).interfaceId), "token does not support IERC1155");
```
Perhaps this will make the code more expressive.
Then we could also add a verification step in the constructor for visibility:
```
    constructor(
        address bridgeAddress
    ) ERCHandlerHelpers(bridgeAddress) {
        require(type(IERC1155).interfaceId == 0xd9b67a26, 'invalid ERC1155 interface');
    }
```